src/SFA.DAS.EmploymentVerification.Database/Tables/ScheduledEmploymentVerification.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE [dbo].[ScheduledEmploymentVerification]
```

Comment:
Is this a duplicate of the tabled defined in ScheduledEC.sql?

Reply:
Removed.